### DISCUSSION TITLE
有没有办法只让 tags 页面显示为两栏？

### DISCUSSION THREAD

**Discussion by GanZhiXiong** [ppoffice/hexo-theme-icarus#937] [Q&A]
因为我第三栏是 tags widget，因此在 tags 页面，第二栏和第三栏重复了，影响视觉。

**Comment by ppoffice** [comment]
不能做到只在tags页面不显示tags挂件，除非修改布局代码。

**Reply by GanZhiXiong** [reply]
> 不能做到只在tags页面不显示tags挂件，除非修改布局代码。

感谢您的回复！😀
看来只能修改布局代码了!

**Reply by GanZhiXiong** [reply]
大佬，您好，能否指点下如何修改代码让 tags 页面 不显示 tags 挂件吗？
